google/cloud/aiplatform/training_jobs.py

```diff
@@ -4157,7 +4164,7 @@ def run(
                 Applies only if [export_evaluated_data_items] is True and
                 [export_evaluated_data_items_bigquery_destination_uri] is specified.
             quantiles (List[float]):
-                Quantiles to use for the `minimize-quantile-loss`
+                Quantiles to use for the ``minimize-quantile-loss``
```

Comment:
Are the double back-ticks intentional?

Reply:
Yes, looking at the generated documentation, the double backtick formats a string as code instead of as a string surrounded by backticks.